bindings/wallet-cordova/src/android/jormungandrwallet/Wallet.java

```diff
@@ -0,0 +1,15 @@
+package com.iohk.jormungandrwallet;
+
+public class Wallet {
+    static {
+        System.loadLibrary("wallet_jni");
+    }
+
+    public native static long recover(String mnemonics);
+
+    public native static void delete(long wallet);
+
+    public native static int totalValue(long wallet);
+
+    public native static long initialFunds(long wallet, byte[] block0);
+}
```

Comment:
I'm not sure if this can be done with cordova, but it feels like this is the `wallet-android` itself already. Can it be imported somehow? or maybe using a symlink or something like that?
Just dropping the idea here.

Reply:
In theory we could just copy the file when packaging, the problem is mostly that I don't know how to use kotlin in the plugin. I think it is not technically impossible (maybe we need to transpile in build time, or maybe can do it when the plugin is installed with a hook).

I agree that it would be nice to have only one file, but I'm not sure if we need to do it now. There are a few differences, though, I'm just using the static methods.

Reply:
Well, we could bring back the java files if they are more useful. Or just keep them both as they are now. Shouldn't be a problem for now since they are pretty simple.

Reply:
I don't think the code repetition is a problem now, but we should be careful about changes in the `wallet-jni` interface, things like adding or removing parameters, changing types, and so on. Maybe we could have some common java wrapper to rely on, and use kotlin on top. 

PD: There is also some repetition for building, as building the plugin is kind of a superset of what's done in wallet-android.